pkg/util/util.go

```diff
@@ -30,6 +30,11 @@ import (
 	"go.uber.org/zap"
 )
 
+var (
+	// ErrorPurgedBinlog indicates binlog was purged
+	ErrorPurgedBinlog = errors.New("binlog was purged")
```

Comment:
This error should belong to the `pump` package?

Reply:
no need, maybe other package need to access it